vita3k/util/include/util/vector_math.h

```diff
@@ -238,35 +241,35 @@ inline SceFloat Vec3<SceFloat>::Normalize() {
 using Vec3f = Vec3<SceFloat>;
 
 template <typename T>
-[[nodiscard]] constexpr decltype(T{} * T{} + T{} * T{}) Dot(const Vec3<T>& a, const Vec3<T>& b) {
+[[nodiscard]] constexpr decltype(T{} * T{} + T{} * T{}) Dot(const Vec3<T> &a, const Vec3<T> &b) {
     return a.x * b.x + a.y * b.y + a.z * b.z;
 }
 
 template <typename T>
-[[nodiscard]] constexpr Vec3<decltype(T{} * T{} - T{} * T{})> Cross(const Vec3<T>& a,
-                                                                    const Vec3<T>& b) {
-    return {a.y * b.z - a.z * b.y, a.z * b.x - a.x * b.z, a.x * b.y - a.y * b.x};
+[[nodiscard]] constexpr Vec3<decltype(T{} * T{} - T{} * T{})> Cross(const Vec3<T> &a,
+    const Vec3<T> &b) {
+    return { a.y * b.z - a.z * b.y, a.z * b.x - a.x * b.z, a.x * b.y - a.y * b.x };
 }
 
 // linear interpolation via SceFloat: 0.0=begin, 1.0=end
 template <typename X>
 [[nodiscard]] constexpr decltype(X{} * SceFloat{} + X{} * SceFloat{})
-    Lerp(const X &begin, const X &end, const SceFloat t) {
+Lerp(const X &begin, const X &end, const SceFloat t) {
     return begin * (1.f - t) + end * t;
 }
 
 // linear interpolation via int: 0=begin, base=end
 template <typename X, int base>
-[[nodiscard]] constexpr decltype((X{} * int{} + X{} * int{}) / base) LerpInt(const X& begin,
-                                                                             const X& end,
-                                                                             const int t) {
```

Comment:
Not sure if the others agree with these changes since it's easier to read before

Reply:
macdu have said is fine in my other pr